docs/kola.md

```diff
@@ -66,6 +66,9 @@ Example format of the file:
     # If no streams are specified, test will be skipped on all streams
     - stream1
     - stream2
+  # The test will only be skipped until this date (will resume on the date)
+  # Format: YYYYMMDD
+  snooze_until: 20210720
```

Comment:
Bikeshed: external tests currently use camelCase for its JSON fields. Should we use that too to be consistent?

Reply:
we could just say `snooze` and drop the `_until` too